python/packages/autogen-agentchat/src/autogen_agentchat/ui/_console.py

```diff
@@ -74,9 +80,57 @@ def notify_event_received(self, request_id: str) -> None:
             event = asyncio.Event()
             self.input_events[request_id] = event
 
+def get_random_ansi_color() -> str:
```

Comment:
I love the simplicty. Though I think we should have some deterministic behavior here. 

How about let's hash the agent name (`source`) of the message into bytes, and then, use the lowest 24 bits to determine the r, g, b numbers?

Reply:
We should also avoid colors that are overly dark or overly white, so lower and upper bound numbers should apply.